src/doc/guide.md

```diff
@@ -414,23 +421,7 @@ We can add the type if we want to, though. Types come after a colon (`:`):
 let x: int = 5;
 ```
 
-If I asked you to read this out loud to the rest of the class, you'd say "`x`
-is a binding with the type `int` and the value `five`."
```

Comment:
My reasoning was that it stuck out like a sore thumb.

Additionally, and more importantly, I think the phrase is wrong. `x` is _not_ a binding. `x` is a variable. It happens to be a _bound_ variable. The variable `x` is bound to the value 5 in this case.

We can discuss and clarify this larger point in normal comments below, if needed.